docs/tutorials/getting-started.md

```diff
@@ -211,14 +211,44 @@ fig
 
 The `<py-repl>` tag creates a REPL component that is rendered to the page as a code editor, allowing you to write executable code inline.
 
-## Visual components
+## The py-config tag
 
-The following tags control visual attributes of your HTML page.
+Use the `<py-config>` tag to set and configure general metadata about your PyScript application in YAML format. If you are unfamiliar with YAML, consider reading [Red Hat's YAML for beginners](https://www.redhat.com/sysadmin/yaml-beginners) guide for more information.
+
+The `<py-config>` tag can be used as follows:
+
+```
+<py-config>
+  - autoclose_loader: false
+  - runtimes:
+    -
+      src: "https://cdn.jsdelivr.net/pyodide/v0.20.0/full/pyodide.js"
+      name: pyodide-0.20
+      lang: python
+</py-config>
+```
+
+The following values are supported by `<py-config>` (all values are optional):
+
+  * autoclose_loader (boolean): If false, PyScript will not close the loading splash screen when the startup operations finish.
+  * name (string): Name of the user application. This field can be any string and is to be used by the application author for their own customization purposes.
+  * version (string): Version of the user application. This field can be any string and is to be used by the application author for their own customization purposes. It is not related to the PyScript version.
+  * runtimes (List of Runtimes): List of runtime configurations. Each Runtime expects the following fields:
+    * src (string): URL to the runtime source.
```

Comment:
We should emphasize that this is required (at least I assume it is), since everything else is being labeled in some way, whether at the top of this list before the colon or below in the other runtime fields.

Another thought, should we just put Required or Optional after every value in the list? Maybe Required is in italics?

Reply:
@fpliger is `src (string)` required for runtimes? We call out the other two as Optional.

Reply:
We could just have two items in those parenthesis, a value type and optional or required.

autoclose_loader (boolean, optional):
name (string, optional):

Etc.

Reply:
We do introduce the table with "The following values are supported by `<py-config>` (all values are optional):"

We could emphasize it more, like:
The following **optional** values are supported by `<py-config>`:

Reply:
Everything in the config is optional. If you specify a run time the source is mandatory. 

I'm fine either adding optional to each one of the single values or as is, not sure if I would explicitly add "mandatory" . Feel free to change as you see fit